src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from '@ecl/ec-react-component-icon/Icon';
+import Logo from '@ecl/ec-react-component-logo/Logo';
+import SearchForm from '@ecl/ec-react-component-search-form/SearchForm';
+
+const SiteHeader = ({
+  logo,
+  languageSelector,
+  searchForm,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, {
+    'ecl-site-header': true,
+  });
+
+  return (
+    <header {...props} className={classNames}>
+      <div className="ecl-site-header__container ecl-container">
+        <div className="ecl-site-header__banner">
+          <a
+            className="ecl-site-header__logo ecl-link ecl-link--standalone"
```

Comment:
Would it have been possible to use the Link component here?

Reply:
link component not suitable for this